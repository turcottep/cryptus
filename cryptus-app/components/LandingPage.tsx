import React from "react";
import NameForm from "./NameForm";
import Navbar from "./Navbar";

export default function LandingPage() {
  return (
    <div className="h-screen relative bg-coquille w-full">
      <div className="h-screen justify-between px-3 md:px-10 2xl:px-20 md:mx-auto flex flex-col items-center">
        <Navbar />

        <div className="flex h-full justify-evenly md:justify-between flex-col w-full items-center md:flex-row">
          <div className="flex justify-between flex-col w-full md:justify-around items-center text-center md:h-full my-4">
            <div className="flex-shrink w-auto md:w-full">
              <h1 className="mx-auto h-auto">
                <img draggable="false" className="w-full" src="title.svg" />
              </h1>
            </div>
            <div className="text-center">
              <p className="md:block leading-normal text-gray-600 text-2xl md:text-3xl xl:text-3xl ">
                <span> The quickest way to show</span>
                <br />
                <span>your NFTs</span>
              </p>
            </div>
            <div className="invisible w-full md:visible md:w-1/2 lg:w-4/5 my-4">
              <NameForm />
            </div>
          </div>
          <div className="flex-shrink w-auto md:w-full min-w-0 min-h-0">
            <img
              className="w-full flex-1 min-w-0 min-h-0"
              draggable="false"
              src="museum.svg"
            />
          </div>
        </div>
        <div className="md:hidden w-full sm:mx-0 md:w-2/5 lg:w-2/5 my-4">
          <NameForm />
        </div>
      </div>
    </div>
  );
}
