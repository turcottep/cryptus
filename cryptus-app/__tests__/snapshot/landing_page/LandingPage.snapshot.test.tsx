// components/__tests__/button.snapshot.test.tsx
import React from "react";
import renderer from "react-test-renderer";
import LandingPage from "../../../components/basic/landing_page/landing_page";

it("renders correctly", () => {
  const tree = renderer.create(<LandingPage />).toJSON();
  expect(tree).toMatchSnapshot();
});
