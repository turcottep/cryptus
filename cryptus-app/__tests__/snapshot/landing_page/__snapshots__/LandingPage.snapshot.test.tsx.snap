// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    <div>
      <div>
        PublicWallet
      </div>
      <div>
        <div
          onClick={[Function]}
        >
          SIGN IN
        </div>
      </div>
    </div>
  </div>
  <div>
    <div>
      <div>
        <img
          src="/images/pw4.png"
        />
      </div>
      <div>
        EASIEST WAY TO
      </div>
      <div>
        TRACK YOUR NFTS
      </div>
      <div>
        <div
          onClick={[Function]}
        >
          TRY NOW
        </div>
      </div>
    </div>
    <div>
      <img
        src="/images/wallet_iphone.png"
      />
    </div>
  </div>
  <div>
    <div>
      <img
        src="/images/market_iphone.png"
      />
    </div>
    <div>
      <div>
        WAKE UP AND CHECK YOUR
      </div>
      <div>
        NETWORTH
      </div>
      <div>
        IN A MATTER OF SECONDS
      </div>
    </div>
  </div>
  <div>
    <div>
      <div>
        SHARE THE ART YOU
      </div>
      <div>
        LOVE
      </div>
      <div>
        WITH THE ONES WHO
      </div>
      <div>
        MATTER MOST
      </div>
    </div>
    <div>
      <img
        src="/images/wallet_iphone.png"
      />
    </div>
  </div>
  <div>
    All Rights Reserved
  </div>
</div>
`;
